### DISCUSSION TITLE
What is the define variable that is defined when PSRAM is enabled?

### DISCUSSION THREAD

**Discussion by XanCraft21** [earlephilhower/arduino-pico#2655] [Q&A] [answered]
Hello, i was going to use the PSRAM capability of the RP2350, and i need to know what the define is called when i enable it. I was going to make a sketch that can work on multiple different processors that support PSRAM, and i was going to use the ifdef or if defined() compile tags to detect which board is being used and if PSRAM is enabled or not. Thank you for your help.

**Comment by earlephilhower** [comment, marked as answer]
https://github.com/earlephilhower/arduino-pico/blob/6024e9a7e82a72e38dd90f42029ba3748835eb2e/cores/rp2040/psram.cpp#L35

It's defined by the board variant in the pins file.

**Reply by XanCraft21** [reply]
Thank you. My project will soon be on its way.
